tests/utils_test.py

```diff
@@ -0,0 +1,22 @@
+"""Unit tests for english language checker"""
+import unittest
+import libcipher.utils as utils
+
+filepath = '/home/travis/build/Chennaipy/libcipher/libcipher/dictionary.txt'
```

Comment:
Would this be a better way to handle the filepath location variable:
filepath = os.path.dirname(os.path.abspath(inspect.getfile(utils)))
Could you tell me the correct way of doing so?
Thank you